lib/tasks/data.rake

```diff
@@ -126,8 +126,8 @@ namespace :data do
           kase.state_machine.flag_for_clearance! press_officer,
                                                  press_office,
                                                  dacu_disclosure
-        rescue Statesman::TransitionFailedError => err
-          puts "!!! transition error received: #{err.message}"
+        rescue  => err
```

Comment:
Since this is catching all exceptions it's going to be hiding backtraces for any error. It may be worth displaying them with:

```
puts err.backtrace.join("\n\t")
```
after all these `puts "!!!...`.